ui/lib/core/addon/templates/components/replication-header.hbs

```diff
@@ -55,14 +40,10 @@
         </ul>
       {{else}}
         <ul>
-          <LinkTo @route="vault.cluster.replication-dr-promote.details" @activeClass="is-active">
+          <LinkTo @route="vault.cluster.replication-dr-promote.details">
             Details
           </LinkTo>
-          <LinkTo
-            @route="vault.cluster.replication-dr-promote"
-            @activeClass="is-active"
-            @current-when="vault.cluster.replication-dr-promote.index"
-          >
+          <LinkTo @route="vault.cluster.replication-dr-promote" @current-when="vault.cluster.replication-dr-promote.index">
             Manage
           </LinkTo>
```

Comment:
I noticed that neither tabs had the active state applied and removing `@activeClass` fixed it.